HostLibraryTests/sample_library.yaml

```diff
@@ -8,7 +8,7 @@
 solutions:
   - name:            'Cijk_Ailk_Bljk_SB_MT128x128x08_K1'
     sizeMapping:
-      globalAccumulation: false
+      globalAccumulation: 0
```

Comment:
I'm not opposed to using numerical IDs to distinguish an algorithm, however I might suggest updating the implementation to use an enum wrapper class for the ID.
To a maintainer, globalAccumulation: GlobalAccumulateAlgo.MyAlgo may be a bit more verbose, but is more explicit and refactor-safe than globalAccumulation: 0, where meaning of 0 must be looked up, and potentially meaning of 0 could change in the future.